### DISCUSSION TITLE
添加谷歌云后不久就失效

### DISCUSSION THREAD

**Discussion by jw8013** [AlistGo/alist#728] [Q&A] [answered]
edu谷歌云盘
程序docker xhofe/alist:latest 即v2.1.3版本

刚添加的时候可以用，但是过了不久就提示： 错误 invalid_grant
重新添加又可以用，过一会儿又失效

INFO[2022-03-11 14:10:16] start server @ 0.0.0.0:5244                  
[GIN] 2022/03/11 - 14:10:36 | 200 |    2.652685ms | *.*.*.* | POST     "/api/public/path"
ERRO[2022-03-11 14:10:42] invalid_grant                                
[GIN] 2022/03/11 - 14:10:42 | 200 |   96.066708ms | *.*.*.* | POST     "/api/public/path"
[GIN] 2022/03/11 - 14:11:22 | 200 |    1.871124ms | *.*.*.* | POST     "/api/public/path"
ERRO[2022-03-11 14:11:23] invalid_grant                                
[GIN] 2022/03/11 - 14:11:23 | 200 |   90.776299ms | *.*.*.* | POST     "/api/public/path"

**Comment by xhofe** [comment]
用的什么client 试试默认的

**Reply by jw8013** [reply]
> 

默认和自建的都一样，个人网盘也炸。。。
RCLONE和OneManager挂载没问题，很久以前挂载的就是了

**Reply by xhofe** [reply]
可以发我个refresh_token测试吗

**Reply by jw8013** [reply]
> 可以发我个refresh_token测试吗

TG里发给你

**Reply by xhofe** [reply]
https://t.me/alist_chat

**Comment by xhofe** [comment, marked as answer]
之前工具token显示错误 现已更改 重新获取一直有效。
